examples/acc-repeated.rs

```diff
@@ -0,0 +1,116 @@
+use binius_circuits::{builder::ConstraintSystemBuilder, unconstrained::unconstrained};
+use binius_core::constraint_system::validate::validate_witness;
+use binius_field::{
+	arch::OptimalUnderlier, packed::set_packed_slice, BinaryField128b, BinaryField1b,
+	BinaryField8b, PackedBinaryField128x1b,
+};
+
+type U = OptimalUnderlier;
+type F128 = BinaryField128b;
+type F8 = BinaryField8b;
+type F1 = BinaryField1b;
+
+// FIXME: Following gadgets are unconstrained. Only for demonstrative purpose, don't use in production
+
+const LOG_SIZE: usize = 8;
+
+// The idea of 'Repeated' column is that one can just copy data from initial column multiple times,
+// so new column is X times bigger than original one. The following gadget operates over bytes, e.g.
+// it creates column with some input bytes written and then creates one more 'Repeated' column
+// where the same bytes are copied multiple times.
+fn bytes_repeat_gadget(builder: &mut ConstraintSystemBuilder<U, F128>) {
+	let bytes = unconstrained::<U, F128, F8>(builder, "input", LOG_SIZE).unwrap();
```

Comment:
These gadgets are missing the push/pop of a namespace

Reply:
Right. This is important, when multiple gadgets are combined into the circuit